github-management/org-owners-guide.md

```diff
@@ -76,6 +78,24 @@ for all orgs going forward.  Notable discrepancies at the moment:
   org. This was a mistake that swapped the usual convention, and we would like
   to rename the team
 
+### Structure and Process
+
+Guidelines on how to create and structure teams are described below:
+
+- Teams should not have any `maintainers` and everyone who should be part of
+the team should be added in the `members` list.
+  - Note that because of how the GiHub API works, org admins still need to be
+    in the `maintainers` list for the teams they are a part of.
+- The `privacy` of a team *must* be `closed`.
+- A new team can be created or a member can be added to a team by
+creating a PR against the [kubernetes/org] repo. The PR must be
+approved by the relevant `OWNERS` or the SIG leads.
+  - For example, addition of a member to `foo-maintainers` must be approved
+    by the `OWNERS` of the repo `foo` or the leads of the SIG associated
+    with the repo.
+- To rename a team, add the `previously: <old-team-name>` field to the team
```

Comment:
I would put this as the first bullet. If someone is renaming a team, it's quicker for them to know this ahead of time before walking through the rest of the instructions.